frontend/src/container/CreateAlertRule/SelectAlertType/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { AlertTypes } from 'types/api/alerts/alertTypes';
+
+import { AlertTypeCard, AlertTypeCards, SelectTypeContainer } from './styles';
+
+function SelectAlertType({ onSelect }: SelectAlertTypeProps): JSX.Element {
+	const { t } = useTranslation(['alerts']);
+	return (
+		<SelectTypeContainer>
+			<h3> {t('choose_alert_type')} </h3>
+			<AlertTypeCards>
+				<AlertTypeCard
```

Comment:
can we iterate over a config to avoid duplicates

Reply:
done